### PR TITLE
Ros2 v0.8.0 autoware planning rviz plugin

### PR DESCRIPTION
**Pull request by tkimura4**
# How to check

1. Open rviz and subscribe topic from display type: Max Velocity

2. publish topic and confirm rviz
3. 
```
ros2 topic pub /planning/scenario_planning/current_max_velocity autoware_planning_msgs/msg/VelocityLimit '{data: 10.0}'
```

### CODE REVIEW

File: common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#include "rviz_common/properties/string_property.hpp"
+#include "rviz_common/tool.hpp"
+#include "rviz_common/validate_floats.hpp"
+
+#include "OgreBillboardSet.h"
+#include "OgreManualObject.h"
+#include "OgreSceneManager.h"
+#include "OgreSceneNode.h"
+
+#include "std_msgs/msg/float32.hpp"
```
Comment (wep21) [inline common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp:40]: autoware_planning_msgs/msg/velocity_limit.hpp?
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/7528c28f5d8b0109a7361a69d1c9f4e67e986ddf/planning/scenario_planning/common/motion_velocity_optimizer/src/motion_velocity_optimizer.cpp#L95

File: common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  void onEnable() override;
+  void subscribe();
+  void unsubscribe();
+
+private Q_SLOTS:
+  void updateTopic();
+  void updateVisualization();
+
+protected:
+  void processMessage(const std_msgs::msg::Float32::ConstSharedPtr msg_ptr);
```
Comment (wep21) [inline common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp:66]: autoware_planning_msgs::msg::VelocityLimit?

File: common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp (hunk 1 above)
Comment (tkimura4) [reply common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp:40]: Thanks! Fixed topic type.

File: common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp (hunk 2 above)
Comment (tkimura4) [reply common/util/rviz_plugins/autoware_planning_rviz_plugin/src/tools/max_velocity.hpp:66]: Thanks! Fixed topic type.

**Comment by wep21** [comment]
looks good.
image

**Pull request merged**